db/migrate/20200624052728_create_frameworks.rb

```diff
@@ -0,0 +1,10 @@
+class CreateFrameworks < ActiveRecord::Migration[6.0]
+  def change
+    create_table :frameworks, id: :uuid do |t|
+      t.string :name, null: false
```

Comment:
maybe name should be unique?

Reply:
Names will be unique per *version*, I can add a check for those 👍

Reply:
added an index/validation scoped to version